src/screens/Transaction.js

```diff
@@ -42,34 +43,34 @@ function TxDetailScreen({ hash }) {
           <Card.HeaderTitle>Transaction Summary</Card.HeaderTitle>
         </Card.Header>
         <Card.Content>
-          <Card.HorizontalContainer>
-            <Card.Column>
+          <Row>
```

Comment:
Not explicitly apart of my desired changes, but the Transaction.js page was busted without this and simply wasn't rendering any content.

Reply:
Yeah this is perfect, my bad for not including this in the other PR